### PR TITLE
Home page fixing version one 

### PR DESCRIPTION
**Pull request by minwwgo**
Home page fixing version one

### CODE REVIEW

**Review by ferhatu — APPROVED** [review]

**Review by 137rx — APPROVED** [review]

File: client/src/App.css [hunk 1]
```diff
@@ -1,7 +1,10 @@
 body {
   background-color: white;
   font-size: 1.5rem;
-  font-family: 'Roboto','sans-serif';
+  font-family: 'Open Sans',
+  sans-serif;
```
Comment (LucyMac) [inline client/src/App.css:5]: Keep this on the same line

File: client/src/App.js [hunk 2]
```diff
@@ -45,16 +54,42 @@ useEffect(()=>{
...
+                {...props}
+                resources={resources}
+                setResources={setResources}
+              />
+            )}
+          />
+          <Route path="/login" component={Login} />
+          <Route path="/contact" component={Contact} />
+
     <BrowserRouter>
```
Comment (LucyMac) [inline client/src/App.js:93]: You're opening 2 `BrowserRouter` but only closing one.

File: client/src/components/Footer.js [hunk 3]
```diff
@@ -1,17 +1,54 @@
...
+              </Nav.Item>
+            </Nav>
+          </Col>
+          <Col  className="d-flex flex-row-reverse">
+            <Nav as="ul" className="text-right">
+              <Nav.Item as="li">
+                <Nav.Link href="/AdminArea">Admin</Nav.Link>
+                <Nav.Link href="/login">login</Nav.Link>
+                <Nav.Link href="/contact">Contact Us</Nav.Link>
+              </Nav.Item>
```
Comment (LucyMac) [inline client/src/components/Footer.js:26]: Each `Nav.Link` should be a `Nav.Item`. There's only one `li` in this list at the moment.

File: client/src/components/Header.js [hunk 4]
```diff
@@ -1,45 +1,45 @@
...
-                Admin
-              </Link>
-            </Nav>
-          </Nav>
-        </Navbar.Collapse>
-      </Navbar>
-    </div>
+    <header className="header">
+      <Row>
+        <Col>
```
Comment (LucyMac) [inline client/src/components/Header.js:10]: Nice use of react-bootstrap here 👍

File: client/src/components/Home/Cards.js [hunk 5]
```diff
@@ -1,38 +1,42 @@
...
-        alt="Card image cap"
-        title="MENTAL HELP"
-        paragraph="If you need help for a mental health crisis, emergency or breakdown, you should get immediate expert advice and assessment. NHS urgent mental health helplines"
-        link="https://www.nhs.uk/using-the-nhs/nhs-services/mental-health-services/where-to-get-urgent-help-for-mental-health/"
-        linkcap="Where to get urgent help"
-      />
-    </div>
+    <section className="small-cards-holder p-5 mb-5">
+      <Container>
+        <Row className="d-flex">
```
Comment (LucyMac) [inline client/src/components/Home/Cards.js:12]: The <Row> component should already be set to display flex, it's one of the main purposes of `Row`

File: client/src/components/Home/Carousel.js [hunk 6]
```diff
@@ -1,39 +1,47 @@
...
-import Carousel from 'react-bootstrap/Carousel'
-import Image from 'react-bootstrap/Image'
+import Carousel from "react-bootstrap/Carousel";
+import Image from "react-bootstrap/Image";
 
-import imageOne from "../../images/image 1.jpg";
+import imageOne from "../../images/image-13.jpg";
 import imageTwo from "../../images/image 2.jpg";
 import imageThree from "../../images/image 3.jpg";
+import { Container, Row,Col } from "react-bootstrap";
```
Comment (LucyMac) [inline client/src/components/Home/Carousel.js:9]: It's best practice to keep external imports (like "react" and "react-bootstrap") at the top of your imports, and have your project's relative imports (like "../../images/image-13.jpg") after.

Also, you can combine all your react-bootstrap imports on one line: 
`import { Container, Row, Col, Carousel, Image } from "react-bootstrap";`

**Review by LucyMac — COMMENTED** [review]
I know this is already merged but I've added some comments anyway :)

**Pull request merged**